### DISCUSSION TITLE
Is it ready for production use?

### DISCUSSION THREAD

**Discussion by mustakimkr** [resend/react-email#627] [Q&A] [answered]
Hello,

First of all, I would like to give a huge shout-out to the React Email team. In this JSX era, building emails with a template engine is a pain. I was desperately looking for a library like this. I'm super excited to use it at work, but as it's still in beta, I would like to know if it's safe to use in production.

Thank you

**Comment by zenorocha** [comment, marked as answer]
There are already tons of companies using React Email. We're definitely ready for production use.

**Reply by mustakimkr** [reply]
Ok. Thank you

**Reply by mustakimkr** [reply]
It will be great if you list who is using React Email somewhere on the website.
